components/index.js

```diff
@@ -0,0 +1,10 @@
+export { default as Button } from './button';
+export { default as Dashicon } from './dashicon';
+export { default as FormToggle } from './form-toggle';
+export { default as IconButton } from './icon-button';
+export { default as Panel } from './panel';
+export { default as Placeholder } from './placeholder';
+export { default as Spinner } from './spinner';
+export { default as Toolbar } from './toolbar';
+
+export { default as withFocusReturn } from './higher-order/with-focus-return';
```

Comment:
Oh I just saw your comment https://github.com/WordPress/gutenberg/pull/929#issuecomment-304771336

I still think we should make a distinction between importing components and HoC.